File: Server/app/Models/Card.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class Card extends Model
{
    protected $fillable = ["uid", "name", "slug", "layout", "rarity", "type", "text", "flavorText", "vitality", "faceNames", "manaCosts", "subtypes", "legalities", "colors", "keywords", "front", "back"];

    protected $hidden = ["id", "created_at", "updated_at"];

    protected $casts = [
        "vitality" => "array",
        "faceNames" => "array",
        "manaCosts" => "array",
        "subtypes" => "array",
        "colors" => "array",
        "keywords" => "array",
        "legalities" => "array",
    ];
}

File: Client/Models/Data/Card.cs
namespace Client.Models.Data
{
    public class Card
    {
        public string UID {get;set;}
        public string Name {get;set;}
        public string Slug {get;set;}
        public string Layout {get;set;}
        public string Rarity {get;set;}
        public string Type {get;set;}
        public string Text {get;set;}
        public string FlavorText {get;set;}
        public Vitality[] Vitality {get;set;}
        public string[] FaceNames {get;set;}
        public string[] ManaCosts {get;set;}
        public string[] Subtypes {get;set;}
        public string[] Colors {get;set;}
        public string[] Keywords {get;set;}
        public string Front {get;set;}
        public string Back {get;set;}
    }
}
